src/call_builder.js

```diff
@@ -163,6 +165,15 @@ export class CallBuilder {
 
     // Temp fix for: https://github.com/stellar/js-stellar-sdk/issues/15
     url.setQuery('c', Math.random());
+    axiosRetry(axios, this.retry);
```

Comment:
adding the retry and the interceptor should be done at init time once, I assume this will add many handlers for each request.
We have an additional issue here, we manipulate axios here in global way,cause axios is reused between `CallBuilder` instances, that means that once you set retry you set to every future instance. 
I think we need to separate axios instances using `axios.create()`.

Reply:
Let's talk about it @yosriz